src/main/java/io/micronaut/build/testresources/MicronautStartTestResourcesServerMojo.java

```diff
@@ -54,21 +56,20 @@
 /**
  * Starts the Micronaut test resources server.
  */
-@Mojo(name = MicronautTestResourcesServerMojo.NAME, requiresDependencyResolution = ResolutionScope.COMPILE_PLUS_RUNTIME)
-public class MicronautTestResourcesServerMojo extends AbstractMojo {
-    public static final String NAME = "start-testresources-server";
+@Mojo(name = MicronautStartTestResourcesServerMojo.NAME, requiresDependencyResolution = ResolutionScope.COMPILE_PLUS_RUNTIME)
+public class MicronautStartTestResourcesServerMojo extends AbstractTestResourcesMojo {
+    public static final String NAME = "start-testresources-service";
 
-    private static final String DEFAULT_ENABLED = "false";
     private static final String DEFAULT_CLASSPATH_INFERENCE = "true";
     private static final String DEFAULT_CLIENT_TIMEOUT = "60";
 
-    private static final String CONFIG_PROPERTY_PREFIX = "micronaut.test-resources.";
+    private static final String TEST_RESOURCES_PROPERTIES = "test-resources.properties";
 
     /**
      * Whether to enable or disable Micronaut test resources support.
      */
-    @Parameter(property =  "micronaut.test-resources.enabled", defaultValue = DEFAULT_ENABLED)
-    private Boolean testResourcesEnabled = Boolean.valueOf(DEFAULT_ENABLED);
+    @Parameter(property =  "micronaut.test-resources.enabled", defaultValue = "false")
```

Comment:
Why was `defaultValue = DEFAULT_ENABLED` inlined?

Reply:
This is basically because I was struggling to understand what Maven was doing and not interpreting my property values, and as part of the process I noticed that it was the only place where such a constant was used.